turbinia/lib/utils.py

```diff
@@ -141,7 +144,14 @@ def bruteforce_password_hashes(password_hashes, timeout=300):
     password_hashes_file_path = fh.name
     fh.write('\n'.join(password_hashes))
 
-  cmd = ['john', password_hashes_file_path]
+  pot_file = os.path.join((tmp_dir or tempfile.gettempdir()), 'hashcat.pot')
+  password_list_file_path = os.path.expanduser('~/password.lst')
```

Comment:
Can we do a check here to verify that this file exists before we try to use it?

Also, If we execute hashcat without the password list, will it be able to use some kind of default or algorithm for this?  I'm wondering what will happen if we try to use Turbinia outside of the Docker container as configured, and it would be nice to have a safe fallback.

Reply:
PTAL. Went back and added the extra John package that comes with its default wordlist.
There is no default or algorithm for hashcat, unfortunately. Once it runs out of dictionary words it won't try an alpha brute force, that'd have to be a separate invocation.
This does mean its run time is naturally very limited.

Reply:
SG, thanks.